pjsip/src/pjsua-lib/pjsua_media.c

```diff
@@ -3612,6 +3616,8 @@ static pj_bool_t is_media_changed(const pjsua_call *call,
         }
 
         /* Compare codec param */
+        if (new_cp == NULL)
```

Comment:
if `pjmedia_stream_info_from_sdp()` returns success, I believe this cannot be NULL.

Reply:
better change to `PJ_ASSERT_RETURN(new_cp, PJ_FALSE);` ?

Reply:
ok